src/screens/block/block.vue

```diff
@@ -87,43 +88,13 @@
         v-if="!isKeyBlock"
         class="block-transactions"
       >
-        <h2
-          v-if="block.height"
-          class="title"
-        >
-          <span class="number">
-            {{ block.transactions.length }}
-          </span> Transaction(s)
-        </h2>
-        <div v-else>
-          <div><FillDummy color="grey" /></div>
-          <div>
-            <FillDummy
-              color="grey"
-              size="big"
-            />
-          </div>
-          <div>
-            <FillDummy
-              color="grey"
-              size="big"
-            />
-          </div>
-          <div>
-            <FillDummy
-              color="grey"
-              size="big"
-            />
-          </div>
-        </div>
         <div
           v-if="block.height"
           class="transactions"
         >
-          <Transaction
-            v-for="t in block.transactions"
-            :key="t.hash"
-            :transaction="t"
+          <MicroBlock
```

Comment:
We do not need to show Micro Block number in this screen.

Reply:
We should only have a list of transactions.

Reply:
updated